__tests__/models/ship.test.js

```diff
@@ -0,0 +1,29 @@
+import Ship from '../../src/models/ship'
+
+describe('Ship Factory', () => {
+  let ship
```

Comment:
For this particular case we can do away with the `beforeEach` and `let` and just declare `ship` as a constant:

```javascript
const ship = Ship(3);
```

Since we're using the same ship in all the tests.